src/main/java/core/basesyntax/util/filereader/FileReaderImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.util.filereader;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String fromFileName) {
+        File file = new File(fromFileName);
```

Comment:
Avoid hardcoding file paths or file names. It's good that you're using a parameter for the file name, but make sure you do not hardcode it anywhere else in the application.